models/user.js

```diff
@@ -9,6 +9,10 @@ const Users = sequelize.define('users', {
     type: Sequelize.DOUBLE,
     defaultValue: null,
   },
+  doneOrDueCount: {
```

Comment:
it feels like we need a better name (and  display label) for this field... but i don't have a suggestion at the moment

Reply:
I couldn't think of anything cleverer here, so I used this to get us started at least.